securedrop_client/gui/widgets.py

```diff
@@ -104,9 +113,17 @@ def setup(self, controller):
 
     def enable_refresh(self):
         self.refresh.enable()
+        self.set_online_style()
```

Comment:
`enable_refresh` was the function triggered when the offline/online mode changes. By making this change to the style and adding it to that function it makes it go beyond its name. So I would propose changing either to change the name to `set_online` (and propagate changes to `gui/main.py`. But I wanted feedback before doing that.

Reply:
I dig it, `set_online` seems like a clear name to me